index.html

```diff
@@ -18,6 +18,14 @@
     </div>
     <nav>
       <img id="burgermenu" src="./img/burger.png" alt="">
+      <div id="menu">
+        <ul id="menu-container">
+          <li class="menu-item"><a class="menu-style" href="#">Portfolio</a></li>
+          <li class="menu-item"><a class="menu-style" href="#">About</a></li>
+          <li class="menu-item"><a class="menu-style" href="#">Contact</a></li>
+          <li class="menu-item"><img id="mail-icon" src="/img/icons/icons8-mail-48.png" alt=""></li>
+        </ul>
+      </div>
```

Comment:
Your headline section doesn't look as the one the Figma Design I see you are using the `height: 100vh;` that cause this please try to adjust the style in order that you page look the same as the one in the Figma design 

- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/165862877-e393cf40-108e-44e9-94ca-cac543195d25.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/165862908-50c19294-f2d0-4746-a3a1-0f16ee7bc5f7.png)